debian/control

```diff
@@ -72,6 +74,7 @@ Build-Depends: cmake,
                qtdeclarative5-ubuntu-content1,
                qtdeclarative5-ubuntu-settings-components (>= 0.11),
                ttf-ubuntu-font-family,
+               ubuntu-system-settings,
```

Comment:
Why is this needed? Are you sure you don't need `libsystemsettings-dev` instead?